R/is_reproducible.R

```diff
@@ -0,0 +1,28 @@
+#' Reproducibility Check for `@code` Slot in `teal_data`
```

Comment:
Needs more work.

Reply:
Extended the documentation a bit in here 38d2b8de0e95712fa60d13e6f347a7d48c3d9b64